apps/skilavottord/web/screens/RecyclingCompanies/RecyclingCompanyUpdate/RecyclingCompanyUpdate.tsx

```
@@ -114,59 +149,46 @@ const RecyclingCompanyUpdate: FC<React.PropsWithChildren<unknown>> = () => {
   }
 
+  const navigateToBaseRoute = () => {
+    const route = isMunicipalityPage
+      ? routes.municipalities.baseRoute
+      : routes.recyclingCompanies.baseRoute
+    return router.push(route)
+  }
+
   const handleUpdateRecyclingPartner = handleSubmit(async (input) => {
     // Not needed to be sent to the backend, causes error if it is sent
     delete input.__typename
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)